docs/release_notes/v1.10.1.md

```diff
@@ -0,0 +1,21 @@
+# Dapr 1.10.1
+
+## Fixes regression in Cloud Event fields "id" and "source"
+
+### Problem
+
+1.10 introduced a regression in pub/sub where publishing a message resulting in the runtime creating a new cloudevent envelope would set the Dapr app-id to the `id` field in the envelope instead of a UUID value, and the `source` field would be set to `"Dapr"` irrespective of the app-id producing the event.
```

Comment:
```suggestion
Dapr 1.10.0 introduced a regression in pub/sub. When publishing a message, Dapr incorrectly sets the Dapr "app-id" as the `id` field in the Cloud Envelope instead of a UUID value, Additionally, the `source` field was be set to `"Dapr"`, irrespective of the app-id producing the event.
```